packages/types/lib/comparator.d.ts

```diff
@@ -0,0 +1,9 @@
+export type StringComparator = '=' | '!=' | '~' | '!~' | '^' | '!^' | '$' | '!$';
+
+export type NumberComparator = '=' | '!=' | '+-' | '>' | '>=' | '<' | '<=';
+
+export type ColorComparator = '=' | '!=' | '+-';
+
+export type JavaScriptComparator = 'matches';
```

Comment:
For what kind of properties used this comparator type?

Reply:
This is used in e.g. `Assert [something] matches [js]`. This is not specifically for properties, but rather a comparator in general - both props and a complete conditions.